rebench/rebench.py

```diff
@@ -124,11 +124,17 @@ def shell_options(self):
             default=False,
             help='Disables execution.'
                  ' It allows to verify the configuration file and other parameters.')
+        execution.add_argument(
+            '-D', '--no-denoise', action='store_true', dest='disable_denoise',
```

Comment:
Yeah, I think it would be nice to keep the flag semantics here, but store the reverse value.
So, keep it as `no-denoise`, but store it into a `use_denoise` flag by using `store_false` and `default=True`